apps/fishing-map/features/workspace/activity/ActivitySchemaFilter.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { MultiSelect, MultiSelectOption, Slider } from '@globalfishingwatch/ui-components'
+import { getPlaceholderBySelections } from 'features/i18n/utils'
+import { SchemaFilter } from 'features/datasets/datasets.utils'
+import styles from './ActivityFilters.module.css'
+
+type ActivitySchemaFilterProps = {
+  schemaFilter: SchemaFilter
+  onSelect: (filterKey: string, selection: MultiSelectOption | MultiSelectOption[]) => void
+  onRemove: (filterKey: string, selection: MultiSelectOption[]) => void
+  onClean: (filterKey: string) => void
+}
+export const showSchemaFilter = (schemaFilter: SchemaFilter) => {
+  return schemaFilter.active && schemaFilter.options.length > 1
+}
+
+function ActivitySchemaFilter({
+  schemaFilter,
+  onSelect,
+  onRemove,
+  onClean,
+}: ActivitySchemaFilterProps): React.ReactElement {
+  const { t } = useTranslation()
+  if (!showSchemaFilter(schemaFilter)) {
+    return null
+  }
+  const { id, tooltip, disabled, options, optionsSelected, type } = schemaFilter
+  if (type === 'number') {
+    const optionValues = options.map(({ id }) => parseInt(id)).sort((a, b) => a - b)
+    const rangeValues =
+      optionsSelected?.length > 0
+        ? optionsSelected.map(({ id }) => parseInt(id)).sort((a, b) => a - b)
+        : optionValues
+
+    const range = [rangeValues[0], rangeValues[rangeValues.length - 1]]
+
+    const onSliderChange = (rangeSelected) => {
+      if (rangeSelected[0] === range[0] && rangeSelected[1] === range[1]) {
+        onClean(id)
+      } else {
+        const selection = rangeSelected.map((id) => ({
+          id: id.toString(),
+          label: id.toString(),
+        }))
+        onSelect(id, selection)
```

Comment:
I would debounce that.

Reply:
Done https://github.com/GlobalFishingWatch/frontend/pull/1203/commits/c3889eb3e9f41907fd0ae7656115c10c27979558